### PR TITLE
Pytave ex recurse2

### PR DESCRIPTION
**Pull request by cbm755**
Here's my fixes, merged against your latest.

### CODE REVIEW

**Comment by genuinelucifer** [comment]
Thanks. I'll merge this. I'll test it in an hour or so..
And then work on other errors.

File: inst/private/store_vars_in_python.m [hunk 1]
```diff
@@ -1,40 +1,29 @@
...
+%varname
+%L
+
+  for i = 1:numel(L)
+    %i
+    x = L{i};
+    if (isa(x, 'sym'));
+      pyobj = pyeval((char(x)));
+      pycall('pystoretemp', pyobj)
+      pyexec([varname '.append(_temp)'])
```
Comment (genuinelucifer) [inline inst/private/store_vars_in_python.m:11]: Great! It really is a nice trick.
.
But, we might still need the python compile unless we deal with the numpy arrays created even for strings by pycall. I'll probably start looking into that now.

Comment (cbm755) [inline inst/private/store_vars_in_python.m:11]: > But, we might still need the python compile unless we deal with the
> numpy arrays created even for strings by pycall. I'll probably start
> looking into that now.

eh?  Please give example what you mean.
- strings work fine
- scalar doubles work fine

Nothing else we need to Symbolic right now I think.

**Comment by cbm755** [comment]
> - strings work fine 

well, with a sufficiently recent pytave

**Comment by genuinelucifer** [comment]
Oh sorry, I hadn't properly seen the code. Yes strings work fine.
.
The sym parts works fine with your fork of pytave and not with
mtmiller/pytave.

**Comment by cbm755** [comment]
> The sym parts works fine with your fork of pytave and not with
> mtmiller/pytave.

Should be easy to avoid the `@pyobject`.
Maybe something like:

```
pyexec (['_temp = ' char(x)])
```

**Comment by genuinelucifer** [comment]
> Should be easy to avoid the `@pyobject`.
> Maybe something like:
> 
> ```
> pyexec (['_temp = ' char(x)])
> ```

Ummm no. We'll need something like `compile` function of python to allow
further operations to be treated on a symbol and not on a string.

**Comment by cbm755** [comment]
> Ummm no.

...!

Please try it first before commenting like that.

**Comment by genuinelucifer** [comment]
> Please try it first before commenting like that.

I'm sorry. I too used this when I first wrote the function to store vars to
python.
I don't know why at all I changed it..
Now that I'm trying it, I didn't find a reason to use `compile`.

**Comment by cbm755** [comment]
Your previous code passed the _string_ in.  My suggestion above generates the appropriate command.

Great that we don't need compile anymore.

**Pull request merged**